charts/observability-pipelines-worker/Chart.yaml

```diff
@@ -13,7 +13,7 @@ icon: https://datadog-live.imgix.net/img/dd_logo_70x75.png
 maintainers:
   - name: Datadog
     email: support@datadoghq.com
-appVersion: "1"
+appVersion: "1.2.0"
```

Comment:
Can technically leave this as-is if you want to change things in one less place. The Datadog Agent also just puts the major version here (it's not functionally used, just documentation)

Reply:
Less updating/they're more strict on semver I'd assume. Vector used this `appVersion` to populate the `image.tag` - but I followed the existing DD chart's pattern so the setup would be more familiar for anyone reviewing PRs from the org.

Reply:
TL;DR is y'alls choice between specificity/accuracy and desire to change one less line per release 😆

Reply:
Ok, in the spirit of following existing practices and since I don't have much expertise here, I'm going to leave this as is.